agent/src/beerocks/slave/son_slave_thread.cpp

```diff
@@ -3678,23 +3679,22 @@ bool slave_thread::slave_fsm(bool &call_slave_select)
             break;
         }
 
-        auto apconfHeader =
-            cmdu_tx.create(0, ieee1905_1::eMessageType::AP_AUTOCONFIGURATION_WSC_MESSAGE);
-        auto tlvAp = cmdu_tx.addClass<wfa_map::tlvApRadioBasicCapabilities>();
-        if (tlvAp == nullptr) {
+        auto c  = cmdu_tx.create(0, ieee1905_1::eMessageType::AP_AUTOCONFIGURATION_WSC_MESSAGE);
+        auto ap = cmdu_tx.addClass<wfa_map::tlvApRadioBasicCapabilities>();
```

Comment:
"ap" is not meaningful name as well.
Please consider to change it to "tlvApRadioBasicCapabilities".

Reply:
This name is way too long.. I will change to radio_basic_caps - we don't care it is a TLV, we just use the class.